pkg/kubelet/apis/podresources/server_v1.go

```diff
@@ -94,3 +96,39 @@ func (p *v1PodResourcesServer) GetAllocatableResources(ctx context.Context, req
 		Memory:  p.memoryProvider.GetAllocatableMemory(),
 	}, nil
 }
+
+// Get returns information about the resources assigned to a specific pod
+func (p *v1PodResourcesServer) Get(ctx context.Context, req *v1.GetPodResourcesRequest) (*v1.GetPodResourcesResponse, error) {
+	metrics.PodResourcesEndpointRequestsTotalCount.WithLabelValues("v1").Inc()
+	metrics.PodResourcesEndpointRequestsGetCount.WithLabelValues("v1").Inc()
+
+	if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.KubeletPodResourcesGet) {
+		metrics.PodResourcesEndpointErrorsGetCount.WithLabelValues("v1").Inc()
+		return nil, fmt.Errorf("Pod Resources API Get method disabled")
```

Comment:
lower case

Reply:
maybe we should keep this as upper case but change it to " PodResources API Get method disabled" same change will apply here as well https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/podresources/server_v1.go#L88